app/controllers/categories_controller.rb

```diff
@@ -41,14 +38,11 @@ def show
   # GET /categories/new
   def new
     @category = Category.new
-    @page_title = "New Category"
   end
 
   # GET /categories/1/edit
   def edit
-    if @category.userid == current_user.id
-      @page_title = "Edit " + @category.name
-    else
+    if @category.userid != current_user.id
       respond_to do |format|
```

Comment:
There were a few like this... I ended up with and empty if block once I removed the `@page_title` declaration, so I reversed the logic of the if statement and got rid of the else